lib/src/main/scala/spinal/lib/bus/wishbone/WishboneSlaveFactory.scala

```diff
@@ -21,31 +21,54 @@ object WishboneSlaveFactory {
   * @param reg_fedback if set to false, the slave will acknowledge as soon as possible otherwise will wait the next clock cycle(default)
   */
 class WishboneSlaveFactory(bus: Wishbone,reg_fedback: Boolean = true) extends BusSlaveFactoryDelayed{
-  bus.DAT_MISO := 0
+
   if(bus.config.isPipelined) bus.STALL := False
 
   val askWrite = bus.isWrite.allowPruning()
   val askRead = bus.isRead.allowPruning()
   val doWrite = bus.doWrite.allowPruning()
   val doRead = bus.doRead.allowPruning()
 
+  val readData = bus.DAT_MISO.clone
+  readData.assignDontCare()
   if(!reg_fedback){
```

Comment:
sorry, I didn't notice last time - is it actually spec-compliant to have a slave where the bus has `bus.config.isPipelined == true` but we configure the WishboneSlaveFactory with `reg_feedback == false`?

Reply:
The spec, as far as I can tell, doesn't say much about an asynchronous ACK specific to pipelined mode. 

It says this:

```
 RULE 3.59
In pipelined mode the MASTER must accept [ACK_I] signals at any time after a
transaction is initiated.
```

and then has this:

```
Even though not initially designed for pipelined mode many simple wishbone slaves
directly support this mode. For this to be true the following characteristics must be true
1. [ACK_O] should be registered
```

so maybe async isn't in spec for pipelined? 

For signalling, it "CYC && STB && !STALL && ACK" is ambiguous but maybe too much to mandate that all master controllers can handle it. 

Whether it's in spec or not, it seems prudent to disallow it here. Should it throw an assertion or just make it registered?